### ISSUE TITLE
Cannot import name 'T' from 're' Fedora 41

### ISSUE THREAD

**Issue by ryleu** [SideStore/SideServer-for-Linux#12]
got this output when trying to run on fedora 41:
```log
$ python3 main.py 
Traceback (most recent call last):
  File "/var/home/ryleu/Documents/ios/SideServer-for-Linux/main.py", line 4, in <module>
    from re import T
ImportError: cannot import name 'T' from 're' (/usr/lib64/python3.13/re/__init__.py)
```
my guess is python 3.13 is not supported. you should state which python version the software targets, or perhaps set up a pyproject.toml that specifies it. poetry is a good tool for this.

**Comment by zafira280** [comment]
@ryleu, do you find a solution?

I am using arch and have the same error.

**Comment by ryleu** [comment]
Yeah, I used an older python version. I think 3.10 worked, but don't quote me on that.

**Comment by tommipontinen76** [comment]
Changing from re import T to import re
And adding this somewhere below fixes it

pattern = re.compile(r"\w+", re.ASCII)

**Comment by nerv0sa** [comment]
I just fixed it by changing T to *

**Comment by Sonys9** [comment]
yea just change T to *
